src/LiveComponent/assets/src/normalize_attributes_for_comparison.ts

```diff
@@ -5,14 +5,14 @@
  * and sets that onto the value attribute. This is useful to compare
  * if two nodes are identical.
  */
-export function normalizeAttributesForComparison(element: HTMLElement): void {
+export function normalizeAttributesForComparison(element: HTMLElement|SVGElement): void {
     if (element.value) {
         element.setAttribute('value', element.value);
     } else if (element.hasAttribute('value')) {
         element.setAttribute('value', '');
     }
 
-    Array.from(element.children).forEach((child: HTMLElement) => {
+    Array.from(element.children).forEach((child: HTMLElement|SVGElement) => {
```

Comment:
I think you can revert these after you follow my instructions about what to do in `onBeforeElUpdated()`.